src/lambda_test_notification_service_formatter.py

```diff
@@ -0,0 +1,59 @@
+import json
+from lib.notification_service.formatter_provider import formatters
+
+
+def lambda_handler(event, context):
+    event_info = json.loads(event)
+    delivery_options_info = event_info.get("delivery_options")
```

Comment:
maybe it also makes sense to check presence of data and delivery_options in event.